apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -385,6 +387,9 @@ export const EventSetupTab = (
 
   const lengthLockedProps = shouldLockDisableProps("length");
   const descriptionLockedProps = shouldLockDisableProps("description");
+  const urlPrefix = orgBranding
```

Comment:
Relying on previos imported `orgBranding` to determine url prefix for event type configuration.